packages/client/src/models/mentor.test.js

```diff
@@ -1,4 +1,4 @@
-import { } from '../providers/FirebaseProvider';
+// import { } from '../providers/FirebaseProvider';
```

Comment:
Can this line be removed?

Reply:
I was not sure whether we were connecting to Firebase, but since we have changed things, I think so?

Reply:
Sounds good.